frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -34,28 +34,40 @@ async function parsePrismaSchema(schemaString: string): Promise<ProcessResult> {
       indices: {},
     }
   }
-
   for (const model of dmmf.datamodel.models) {
     for (const field of model.fields) {
-      if (
-        field.relationName &&
-        (field.relationFromFields?.length ?? 0) > 0 &&
+      if (!field.relationName) continue
+
+      const existingRelationship = relationships[field.relationName]
+      const isTargetField =
+        field.relationToFields?.[0] &&
         (field.relationToFields?.length ?? 0) > 0 &&
         field.relationFromFields?.[0] &&
-        field.relationToFields?.[0]
-      ) {
-        const relationship: Relationship = {
-          name: field.relationName,
-          primaryTableName: field.type,
-          primaryColumnName: field.relationToFields[0],
-          foreignTableName: model.name,
-          foreignColumnName: field.relationFromFields[0],
-          cardinality: 'ONE_TO_MANY',
-          updateConstraint: 'NO_ACTION',
-          deleteConstraint: 'NO_ACTION',
-        }
-        relationships[relationship.name] = relationship
-      }
+        (field.relationFromFields?.length ?? 0) > 0
+
+      const relationship: Relationship = isTargetField
```

Comment:
Ah, I think `as const` can remove this type expression.

```suggestion
      const relationship = isTargetField
```